chirp/drivers/ft450d.py

```diff
@@ -1489,3 +1674,19 @@ def set_settings(self, settings):
                 except Exception:
                     LOG.debug(element.get_name())
                     raise
+
+@directory.register
+class FT450Radio(FT450DRadio):
+    """Yaesu FT-450"""
+    FT450 = True
+    MODEL = "FT-450"
+
+    @classmethod
+    def match_model(cls, filedata, filename):
```

Comment:
Since you're adding a new class, just make this `return False`. For several years now, CHIRP appends a metadata blob to the end of the images to do deterministic detection of models. Drivers from before that need to still detect the old-style images, but no driver/model after that point need to. If you just `return False` here, you'll be saying "only ever detect an FT-450 image by metadata" which means your driver and images won't be confused for FT-450D ones.

There are lots of other examples of this in the code, but here's one for reference:

https://github.com/kk7ds/chirp/blob/master/chirp/drivers/btech.py#L4065-L4066